Sources/AblyAssetTrackingInternal/Configuration/ConnectionConfiguration+Extensions.swift

```diff
@@ -18,6 +18,9 @@ extension ConnectionConfiguration {
         } else {
             clientOptions.key = apiKey
         }
+        clientOptions.addAgent("ably-asset-tracking-swift", version: Version.libraryVersion)
+        clientOptions.clientId = clientId
```

Comment:
What's this `clientId` line doing? It seems we already do this earlier on in the method.

Reply:
d6b7fbeccc4cebb914142829f1017b4a978936c1